lib/portlayer/exec2/remote/portlayer_rpc_client.go

```diff
@@ -0,0 +1,134 @@
+// Copyright 2016 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package remote
+
+import (
+	"net/rpc"
+
+	"github.com/vmware/vic/lib/portlayer/exec2"
+)
+
+type PortLayerRPCClient struct {
+}
+
+var serverAddress = "localhost"
+var client *rpc.Client
+
+func (*PortLayerRPCClient) Connect() error {
```

Comment:
Absolutely not. This is purely to demonstrate how the API could be exposed via RPC. This serves the following purposes:
1) To allow the port layer to be tested and driven directly (for function testing etc)
2) To allow for something like TCE to be developed. A consumer of the PL may choose to export it an any number of ways and it should be amenable to that.
3) To demonstrate how the handle parameter can be rich and sparse using the same interface